db/migrate/20160114200145_add_mirror_on_sync_to_repositories.rb

```diff
@@ -0,0 +1,6 @@
+class AddMirrorOnSyncToRepositories < ActiveRecord::Migration
+  def change
+    add_column :katello_repositories, :mirror_on_sync, :boolean, :default => true, :null => false
+    update "UPDATE katello_repositories SET mirror_on_sync = 'f'"
```

Comment:
Yeah it shouldn't cause a save in pulp anymore (hasn't for some time).  You may want to use http://api.rubyonrails.org/classes/ActiveRecord/Relation.html#method-i-update_all  in order to do it with a similar sql query, you'd need to redefine the model.   I'm okay doing it this way but would probably prefer Repository.update_all